ftw/keywordwidget/profiles/uninstall_plone5/registry.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<registry>
+
+    <records prefix="plone.resources/keywordwidget_javascript"
+             interface='Products.CMFPlone.interfaces.IResourceRegistry'
```

Comment:
Could we uniformly use the same apostrophe in the entire file?

Reply:
👍